nfiles.lisp

```diff
@@ -88,9 +88,44 @@ methods. See `resolve', `serialize', etc.
 
 The `name' slot can be used to refer to `file' objects in a human-readable fashion."))
 
+(defun try-restart (name &rest arguments)
```

Comment:
Maybe use `macrolet` to generate those instead? Feels too repetitive...

Reply:
Thought about it, but this makes "go to definition" less meaningful.  That said, for such a trivial function, maybe it's worth it.

Reply:
Still, "go to definition" would show the full macrolet, and the pattern would be apparent enough.

But I'll leave it to your judgment here :)